AirCasting/SessionViews/ABMeasurementsView.swift

```diff
@@ -8,6 +8,7 @@ struct ABMeasurementsView: View {
     @ObservedObject var session: SessionEntity
     var threshold: SensorThreshold
     @Binding var selectedStream: MeasurementStreamEntity?
+    let showValues: Bool
```

Comment:
`Bool` flags are usually not the best for such purposes. It's clunky and slightly beclouds the call site. Much better and future-proof option (even for such binary situations) would be a dedicated enum:

```Swift
enum MeasurementPresentationStyle {
    case showValues
    case hideValues
}
```